pkg/export/export.go

```diff
@@ -475,12 +484,66 @@ func (e *Exporter) triggerNext() {
 	}
 }
 
-// ClientName and Version are used to identify to User Agent. TODO(maxamin): automate versioning.
 const (
+	// ClientName is used to identify the User Agent.
 	ClientName = "prometheus-engine-export"
-	Version    = "0.6.3"
+	// Align with git tag.
+	// TODO(TheSpiritXIII): Remove with https://github.com/golang/go/issues/50603
+	exportVersion = "v0.7.0-rc.0"
+	// Align with go.mod.
+	moduleName = "github.com/GoogleCloudPlatform/prometheus-engine"
 )
 
+// Testing returns true if running within a unit test.
+// TODO(TheSpiritXIII): Replace with https://github.com/golang/go/issues/52600
+func Testing() bool {
+	return flag.Lookup("test.v") != nil
+}
+
+// Version is used in the User Agent.
```

Comment:
Can we add more commentary on behavior of version function? Sounds like it does a lot depending if it's rule evaluator, prometheus engine or run in testing mode?  It's not trivial.

Reply:
Done!